### DISCUSSION TITLE
Zoom Hosts

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/projectpythia.github.io#53] [General]
I was unable to add any Project Pythia members as an "Alternative Host" (which means you would be the host if I did not show up for the meeting because none of you are a user on the Zoom account. I am still able to assign you as a co-host once the meeting has already begun but not before.

The good news is that I was able to change the default screen sharing setting so that all participants can share their screen during a meeting, given someone else isn't already sharing. I might have to change this setting a few times (before hosting tutorials and back), but things should now be set up to run smoothly even if I could not attend the meeting.

Were there any other features of being a host that we needed?

**Comment by clyne** [comment]
Bummer. Thanks for checking.
